assets/maps/prefabs/prefab_safehouse.dmm

```diff
@@ -0,0 +1,164 @@
+"np" = (/obj/stool/bed{pixel_y = 11},/obj/stool/bed,/obj/item/clothing/suit/bedsheet/royal{pixel_y = 11},/obj/item/clothing/suit/bedsheet/royal,/obj/machinery/light{brightness = 1.4; dir = 4; layer = 9.1; name = "worn light fixture"; pixel_x = 10},/turf/simulated/floor/carpet/red/fancy/edge/se,/area/prefab/safehouse)
+"ns" = (/obj/stool/chair/couch/red{dir = 5},/turf/simulated/floor/carpet/red/fancy/narrow/se,/area/prefab/safehouse)
+"nz" = (/obj/decal/fakeobjects/pipe{desc = "Part of the area's air supply system."; dir = 10},/obj/decal/cleanable/dirt/dirt5,/turf/simulated/floor/plating,/area/prefab/safehouse)
+"nF" = (/obj/reagent_dispensers/fueltank,/turf/simulated/floor/airless,/area/prefab/safehouse)
+"nK" = (/obj/decal/cleanable/dirt/dirt4,/turf/simulated/floor/specialroom/bar,/area/prefab/safehouse)
+"nQ" = (/obj/storage/crate/wooden{layer = 2.9},/obj/item/reagent_containers/food/drinks/bottle/hobo_wine{pixel_x = -2; pixel_y = -2},/obj/item/reagent_containers/food/drinks/bottle/hobo_wine{pixel_x = -5},/obj/item/reagent_containers/food/drinks/bottle/hobo_wine{pixel_x = 7},/obj/item/reagent_containers/food/drinks/bottle/hobo_wine{pixel_x = 1},/obj/decal/cleanable/dirt,/turf/simulated/floor/plating,/area/prefab/safehouse)
+"nW" = (/obj/item/reagent_containers/pill/cyberpunk{pixel_x = 2; pixel_y = -8},/obj/decal/cleanable/vomit{pixel_x = -1; pixel_y = 5},/turf/simulated/floor/plating,/area/prefab/safehouse)
+"or" = (/obj/machinery/door/poddoor{id = "clone room"; name = "lab doors"},/turf/simulated/floor/neutral,/area/prefab/safehouse)
+"ot" = (/obj/table/reinforced/auto,/obj/item/scalpel,/obj/item/circular_saw{pixel_x = 3; pixel_y = 12},/obj/item/staple_gun,/obj/item/hemostat,/obj/item/surgical_spoon,/obj/item/suture,/obj/item/reagent_containers/syringe/calomel,/obj/item/implanter{pixel_x = -5},/obj/machinery/light{brightness = 1.4; dir = 4; layer = 9.1; name = "worn light fixture"; pixel_x = 10},/turf/simulated/floor/bluewhite{dir = 6},/area/prefab/safehouse)
+"ou" = (/obj/submachine/chef_oven,/obj/decal/tile_edge/stripe/extra_big,/turf/simulated/floor/black,/area/prefab/safehouse)
+"ox" = (/obj/machinery/chem_dispenser/alcohol,/obj/decal/tile_edge/stripe/extra_big,/obj/machinery/light{brightness = 1.4; dir = 1; layer = 9.1; name = "worn light fixture"; pixel_y = 14},/turf/simulated/floor/black,/area/prefab/safehouse)
+"oM" = (/obj/decal/cleanable/dirt,/turf/simulated/floor/wood,/area/prefab/safehouse)
+"oY" = (/obj/item/storage/wall/medical{dir = 1; pixel_y = 32},/turf/simulated/floor/bluewhite{dir = 1},/area/prefab/safehouse)
+"pl" = (/obj/decal/cleanable/dirt,/turf/simulated/floor/carpet/red/standard/narrow/west{dir = 1},/area/prefab/safehouse)
+"pA" = (/obj/table/auto,/obj/item/storage/toolbox/mechanical,/turf/simulated/floor/plating,/area/prefab/safehouse)
+"pJ" = (/obj/machinery/chem_dispenser,/obj/decal/tile_edge/stripe/extra_big,/turf/simulated/floor/black,/area/prefab/safehouse)
```

Comment:
Please replace with a "busted dispenser" fakeobj

Reply:
Required for puzzle, will have a think about alternatives